scripts/lb-wrapper

```diff
@@ -26,11 +20,13 @@ log() {
     echo "$(date +'%Y-%m-%d %H:%M:%S') - [$level] $message" | tee -a ${LOG_FILE}
 }
 
-if [ $# -eq 0 ]; then
-    log "Error" "No arguments provided. Please provide a URL to download."
+if [ $# -lt 2 ]; then
```

Comment:
Consider enforcing exactly 2 input parameters?

(And adjust the error msg if so!)

```suggestion
if [ $# -ne 2 ]; then
```